packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -58,21 +59,27 @@ export class ActiveWorkflowRunner {
 
 		this.activeWorkflows = new ActiveWorkflows();
 
+		const logger = (global as any).logger as ILogger;  // tslint:disable-line:no-any
```

Comment:
Can we not do that 1x outside of the class so that we do not have this line multiple times in the same file and can simply use directly `logger` everywhere inside of it?